src/macros.jl

```diff
@@ -503,6 +503,28 @@ and
 @transform(df, a = :x, b = :y)
 ```
 
+`@transform` uses the syntax `@byrow` to wrap transformations in
+the `ByRow` function wrapper from DataFrames, enabling broadcasting
+and more. For example, the call
+
+```
+@transform(df, y = @byrow :x == 1 ? "true" : "false)
+```
+
+becomes
+
+```
+transform(df, :x => ByRow(x -> x == 1 ? "true", "false") => :y)
+```
+
+a transformation which  cannot be conveniently expressed
```

Comment:
```suggestion
a transformation which cannot be conveniently expressed
```
BTW, is this still the case on Julia 1.7? Otherwise I wouldn't mention it.